apple/macbook-pro/12-1/README.md

```diff
@@ -0,0 +1,14 @@
+# MacBook Pro 12,1
+
+## Wireless won't get reconnected after resume/hibernate
+
+As an example similar code could be used to restart wireless interface:
```

Comment:
```suggestion
The wifi driver is unloaded before suspend/hibernate to workaround driver issues.
This means it might be required to restart your wifi deamon i.e. wpa_supplicant:
```